src/sql/parts/dashboard/common/dashboardPanel.css

```diff
@@ -9,6 +9,10 @@ panel.dashboard-panel > .tabbedPanel {
 	box-sizing: border-box;
 }
 
+panel.dashboard-panel > .tabbedPanel .tabList .tab .tabLabel {
+	text-transform: none;
```

Comment:
Override the text transform to uppercase

Reply:
Are we sure this is what we want? The reason it does that is because it was meant to mimic the output viewlet in terms of its look and feel
![image](https://user-images.githubusercontent.com/4324725/37734564-ca3ac3c2-2d08-11e8-8775-ecfcb6e52a62.png)
![image](https://user-images.githubusercontent.com/4324725/37734566-ceb739b2-2d08-11e8-8d82-feb91aef913f.png)

Reply:
This brings back memories.  Please read through some comments on this blog for context https://blogs.msdn.microsoft.com/visualstudio/2012/06/05/a-design-with-all-caps/...
...personally no preference on the UX, but just some feedback that some people REALLY CARE ABOUT THIS 😄

Reply:
I don't particularly care either, but I think consistency is important and look and feel. Vscode has already made the decision to use uppercase for titles (see viewlet titles, output window, explorer pane titles, etc).

Reply:
@anthonydresser I'll let you work this one out with the UX folks, just found the topic amusing...I think that's the most commented blog in Visual Studio history.

Reply:
We had a conversation offline and agree to keep it all caps for tab's title